conf/resources/kindsOfPackage.json

```diff
@@ -0,0 +1,2578 @@
+[
+  {
+    "state": "valid",
+    "activeFrom": "2015-10-01",
+    "code": "1A",
+    "description": "Drum, steel",
+    "type" : "Other"
```

Comment:
Just need to be careful here as I don't believe this is how the data will get served to us? Can we safely remove the `type` values?

Reply:
Copy and paste from the old repo. Will remove.